### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rkalytyn**

### CODE REVIEW

File: src/checkPassword.test.js [hunk 1]
```diff
@@ -8,12 +8,38 @@ describe(`Function 'checkPassword':`, () => {
...
+  });
 
+  it(`should return 'false' for password shorter than 8 characters`, () => {
+    expect(checkPassword('Pass!1')).toBe(false);
   });
 
-  // write more tests here
+  it(`should return 'false' for password longer than 16 characters`, () => {
+    expect(checkPassword('P@ssword.pass!123')).toBe(false);
+  });
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: check all boundary value

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
Check the comments, please

**Review by Julia14012000 — APPROVED** [review]
Good job!
